src/main/java/com/jumani/rutaseg/domain/Order.java

```diff
@@ -86,5 +99,39 @@ public Long getClientId() {
     private Order() {
     }
 
+    public void update(boolean pema, boolean port, boolean transport,
+                       ArrivalData arrivalData, DriverData driverData,
+                       CustomsData customsData, List<Container> containers, ConsigneeData consignee) {
+
+
+        if (pema) {
+            this.pema = pema;
+        }
+
+        if (port) {
+            this.port = port;
+        }
 
+        if (transport) {
+            this.transport = transport;
+        }
+
+        if (arrivalData != null) {
+            this.arrivalData = arrivalData;
+        }
+
+        if (driverData != null) {
+            this.driverData = driverData;
+        }
+
+        if (customsData != null) {
+            this.customsData = customsData;
+        }
+
+        this.containers = containers != null ? containers : Collections.emptyList();
```

Comment:
acá nunca te va a venir nulo, creo que lo más fácil es asginarlo directamente: this.containers = containers